src/lib/stores/filter_store.ts

```diff
@@ -0,0 +1,28 @@
+import { writable } from 'svelte/store';
+
+import type { Filters } from 'src/types/filters.type';
+
+import { FilterRepository } from '../repositories/filter_repository';
+import { filterListener } from './filtered_items_store';
+
+const currentFilter: Filters = {
+	easy: false,
+	search: ''
+};
```

Comment:
I'd rename this to being all caps and to `DEFAULT_FILTER`. The thing is, we don't want to store the filters in a global object. If you do `get(filterStore)` you can get the latest value that the FilterRepository has sent to the store, so it replaces this nicely.